pymatgen/core/trajectory.py

```diff
@@ -12,10 +12,12 @@
 from typing import Any, Union
 
 import numpy as np
+from ase.io.trajectory import Trajectory as ASE_Trajectory
```

Comment:
ASE is not a core requirement of pymatgen, so you can't do this.

Reply:
Oops... Is there an alternative option for this?

Reply:
Yes, move the import statement to this line: https://github.com/materialsproject/pymatgen/pull/3422/files#diff-4330101375652bb97d6774e06676d30a6c51537d6df49f01eba66863561839d9R549, within the function and immediately abovve where it's being called. You may also wish to wrap it within a `try`/`except ImportError` block to raise a helpful error message if the import statement does not work.